src/ert/config/_get_num_cpu.py

```diff
@@ -110,14 +110,10 @@ def _get_num_cpu(
             keyword = next(words, None)
             keyword = keyword[0 : min(len(keyword), 8)] if keyword is not None else None
             if keyword == "TITLE":
-                # Skip two non-blank lines following a TITLE
-                for _ in range(2):
-                    line: list[str] = []
-                    while line == []:
-                        nline = parser.next_line(None)
-                        if nline is None:
-                            break
-                        line = list(nline)
+                # Skip one line following a TITLE
+                for _ in range(1):
```

Comment:
Can't this line with `for` just be dropped?

Reply:
fixed